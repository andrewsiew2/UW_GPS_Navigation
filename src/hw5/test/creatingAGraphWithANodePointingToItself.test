# Test if a node can point to itself

# Create a graph
CreateGraph graphWithANode

# Add a node
AddNode graphWithANode NodeA

# Add an edge
AddEdge graphWithANode NodeA NodeA label

# Print the nodes in the graph and the outgoing edges
ListNodes graphWithANode
ListChildren graphWithANode NodeA