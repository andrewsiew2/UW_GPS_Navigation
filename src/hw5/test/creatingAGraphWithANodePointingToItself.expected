# Test if a node can point to itself

# Create a graph
created graph graphWithANode

# Add a node
added node NodeA to graphWithANode

# Add an edge
added edge label from NodeA to NodeA in graphWithANode

# Print the nodes in the graph and the outgoing edges 
graphWithANode contains: NodeA 
the children of NodeA in graphWithANode are: NodeA(label)